lunardate/__init__.py

```diff
@@ -277,3 +277,29 @@ def from_days(days):
 
         return LunarDate(start_year + year_idx, target_month + 1,
                          days + 1, is_leap_month)
+
+    @staticmethod
+    def from_lunar(year, month, day, is_leap_month = False):
```

Comment:
how about change name from (from_lunar) to (to_solar)?
I think it is easy to understand.